stdnum/li/__init__.py

```diff
@@ -19,3 +19,6 @@
 # 02110-1301 USA
 
 """Collection of Liechtenstein numbers."""
+
+# provide aliases
+from stdnum.li import peid as vat  # noqa: F401
```

Comment:
According to https://www.vatify.eu/liechtenstein-vat-number.html the MWST is issued for VAT purposes.